docs/reference/sampler_composites/api.rst

```diff
@@ -190,6 +197,32 @@ Mixin Methods
    Initialized.parse_initial_states
 
 
+Creating a Scoped Sampler
```

Comment:
just sampler or also composite? Or is that implicit in "composed sampler"?

Reply:
I followed the pattern in this file. `Composite` can also be `Structured` or `Initialized`.